x-pack/platform/plugins/shared/fleet/server/services/epm/packages/cache.ts

```diff
@@ -26,6 +27,10 @@ class CacheSession {
 
   private _agentTemplateAssetsMap?: LRUCache<string, PackagePolicyAssetsMap>;
 
+  private _handlebarsCompiledTemplate?: LRUCache<string, HandlebarsTemplateDelegate>;
```

Comment:
we should check how much memory does this cache use when full

Reply:
It seems pretty negligible, when running aws upgrade, that is our package with the most of agent template the memory usage was almost the same as before.